### ISSUE TITLE
DELETE with JSON Body doesn't work

### ISSUE THREAD

**Issue by Cir0X** [balderdashy/sails#3146]
According to 
http://sailsjs.org/documentation/reference/blueprint-api/destroy#?route
it should be possible to send **DELETE** requests with JSON body.

But instead of getting the deleted item, you'll get an **400 - Bad Request** and sails throws:

`Error: No `id` parameter provided.(Note: even if the model's primary key is not named `id`- `id`should be used as the name of the parameter- it will be mapped to the proper primary key name)`

Occurred in v0.11

**Comment by sailsbot** [comment]
Thanks for posting, @Cir0X.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
